tests/commands/publish_tests.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from courseraprogramming.commands import publish
+
+from mock import ANY, call, MagicMock, patch
+
+
+class PublishParams:
+    get_endpoint = "fake-endpoint"
+    publish_endpoint = "fake-endpoint"
+    publish_action = "fake-action"
+    course = "2345"
+    item = "3456"
+    quiet = 1
+
+
+def fake_get_metadata(oauth, get_endpoint, course_id, item_id):
+    return {
+        "version": 0,
+        "authorId": 1234,
+        "publishedAssignmentId": 4321,
+        "updatedAt": 10000000,
+    }
+
```

Comment:
Could you add a parser test to make sure command line parsing works how you expect it to?